### DISCUSSION TITLE
A component that behaves like HoverCard on desktop and Popover on mobile

### DISCUSSION THREAD

**Discussion by lcswillems** [chakra-ui/chakra-ui#9075] [Ideas]
### Description

The HoverCard component doesn't make much sense to me as it only works on desktop. What's the point of having a component that only works on desktop? It seems useless to me but maybe I'm missing something?

On the contrary, the Popover component is great on mobile but not ideal on desktop as it requires an extra click.

It would be great to have a component that triggers on hover on desktop and that triggers on click on mobile.

### Problem Statement/Justification

The HoverCard component doesn't make much sense to me as it only works on desktop. What's the point of having a component that only works on desktop? It seems useless to me but maybe I'm missing something?

On the contrary, the Popover component is great on mobile but not ideal on desktop as it requires an extra click.

### Proposed Solution or API

It would be great to have a component that triggers on hover on desktop and that triggers on click on mobile.

**Comment by segunadebayo** [comment]
Feel free to make this in your userland code.

We’re not looking to add this Chakra at the moment. 

You can make something decent by composing the HoverCard and Popover components into some form of “AdaptiveHovercard”

**Comment by segunadebayo** [comment]
Compose HoverCard and Popover in your app. We are not adding a third primitive for this.
